app/src/index.js

```diff
@@ -2,11 +2,24 @@
 /// user with a visual representation of this process (welcome screen). It also implements a view
 /// allowing to choose a debug rendering test from.
 
+var is_electron = window && window.process && window.process.type;
+var static_path;
+
+if (is_electron) {
+    // To avoid compile-time evaluation
+    eval("window.NODE_ENV=process.env.NODE_ENV; window.DIR_NAME = __dirname");
```

Comment:
who will compile-time evaluate it? This sounds like a terrible hack tbh. Are there any docs telling to do it this way?